### PR TITLE
docs: add OpenAPI input as example

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
This PR adds OpenAPI input as an example.

### CODE REVIEW

**Review by magicmatatjahu — COMMENTED** [review]
Only one suggestion :)

File: README.md [hunk 1]
```diff
@@ -66,16 +66,16 @@ To see the complete feature list for each language, please click the individual
...
   <tr>
-    <td><a href="./docs/usage.md">JSON Schema</a></td>
+    <td><a href="./docs/usage.md#generate-models-from-json-schema-documents">JSON Schema</a></td>
     <td>We support the following JSON Schema versions: <em>Draft-4, Draft-6 and Draft-7</em></td>
   </tr>
   <tr>
-    <td><a href="./docs/usage.md">OpenAPI</a></td>
-    <td>We support the following OpenAPI versions: <em>Swagger 2.0 and OpenAPI 3.0</em>, which generates models for all the defined path request and responses.</td>
+    <td><a href="./docs/usage.md#generate-models-from-swagger-20-documents">OpenAPI</a></td>
+    <td>We support the following OpenAPI versions: <em>Swagger 2.0 and OpenAPI 3.0</em>, which generates models for all the defined request and response payloads.</td>
```
[the remark this answers is not in the document]
Comment (jonaslagoni) [reply README.md:78]: Good call, changed 🙂

**Review by magicmatatjahu — APPROVED** [review]
LGTM! 🚀

**Pull request merged**